qt/README.md

```diff
@@ -0,0 +1,15 @@
+## Running locally from source
+
+Install the dependencies and run the `application.py` script:
+```bash
+$ pip install -r requirements.txt
```

Comment:
PySide is not a requirement. Maybe we should make this an extras requirement, since I don't really want to have Qt as a requirement for nwbwidgets in general

Reply:
yeah I agree, for that I created a separate `requirements.txt` inside the `/qt` folder, which contains the necessary packages to run this